tests/python/unittest/test_numpy_op.py

```diff
@@ -4846,7 +4846,7 @@ def forward(self, param1, param2):
     dtypes = ['float16', 'float32', 'float64']
     op_names = ['uniform_n', 'normal_n']
 
-    for bshape, eshape, dtype, op in itertools.product(batch_shapes, event_shapes, dtypes, op_names):
+    for bshape, eshape, _, op in itertools.product(batch_shapes, event_shapes, dtypes, op_names):
```

Comment:
dtype should be added to line 4834 and 4842 so that different data types are actually tested.